operations/image_management/Import_External_Image_to_IMS.md

```diff
@@ -7,66 +7,63 @@ the following procedure can be used to add this external image to IMS and upload
-* Images in the .txz compressed format need to be converted to SquashFS in order to use IMS image customization.
+* Images in the `.txz` compressed format need to be converted to SquashFS in order to use IMS image customization.
 
 ## Procedure
 
-<a name="ensure_supported_format"></a>
-1.  Ensure that the image root is in a supported format.
+1. <a name="ensure_supported_format"></a>Ensure that the image root is in a supported format.
 
     IMS requires that an image's root filesystem is in SquashFS format. Select one of the following options based on the current state of the image root being used:
 
     * If the image being added is in `tgz` format, refer to [Convert TGZ Archives to SquashFS Images](Convert_TGZ_Archives_to_SquashFS_Images.md).
-    * If the image being added meets the above requirements, proceed to [Create an IMS Image Record](#create_image_record).
+    * If the image being added meets the above requirements, proceed to [Create an IMS image record](#create_image_record).
     * If the image root is in a format other than `tgz` or SquashFS, convert the image root to `tgz`/SquashFS before continuing.
 
-    <a name="create_image_record"></a>
-1.  Choose a descriptive name for the new IMS Image Record.
+1. <a name="create_image_record"></a>Choose a descriptive name for the new IMS image record.
 
     Set the `IMS_ROOTFS_FILENAME` variable to the chosen name.
 
     ```bash
     ncn# IMS_ROOTFS_FILENAME=sles_15_image.squashfs
     ```
 
-1.  Create a new IMS image record for the image.
+1. Create a new IMS image record for the image.
 
     ```bash
     ncn# cray ims images create --name $IMS_ROOTFS_FILENAME
     ```
 
     Example output:
 
-    ```
+    ```toml
     created = "2018-12-04T17:25:52.482514+00:00"
     id = "4e78488d-4d92-4675-9d83-97adfc17cb19"
     name = "sles_15_image.squashfs"
     ```
 
-1.  Create an environment variable for the ID of the new IMS Image Record.
+1. Create an environment variable for the ID of the new IMS image record.
 
-    Set and export the `IMS_IMAGE_ID` variable, using the `id` field value from the returned data in the previous step.
+    Set the `IMS_IMAGE_ID` variable, using the `id` field value from the returned data in the previous step.
 
     ```bash
-    ncn# export IMS_IMAGE_ID=4e78488d-4d92-4675-9d83-97adfc17cb19
+    ncn# IMS_IMAGE_ID=4e78488d-4d92-4675-9d83-97adfc17cb19
     ```
 
-    <a name="upload_to_s3"></a>
-1.  Upload the image root to S3.
+1. <a name="upload_to_s3"></a>Upload the image root to S3.
 
-    1.  Set an environment variable with the actual path and filename of the image root to be uploaded.
+    1. Set an environment variable with the actual path and filename of the image root to be uploaded.
 
-        > This may be the same name as your `IMS_ROOTFS_FILENAME`, but it is not required to be.
+        > This may be the same name as `IMS_ROOTFS_FILENAME`, but it is not required to be.
 
         ```bash
         ncn# ROOTFS_FILENAME=/home/rootfs.squashfs
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815